docker/Dockerfile

```
@@ -6,26 +6,19 @@
 # BUILD: docker build --network=host --file docker/Dockerfile --tag pynvml .
 # RUN: docker run --gpus all --rm -it  pynvml
 ##############################################################
-FROM python:3.6
+FROM python:3.8
 
 ENV DEBIAN_FRONTEND noninteractive 
 
-ADD ./requirements.txt /app/requirements.txt
-
-RUN pip install --no-cache-dir --upgrade pip && \
-    pip install --no-cache-dir -r /app/requirements.txt && \
+ADD . /app/.
+RUN cd /app && \
+    pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir . && \
```

Think about it more, maybe we can do a development install using `-e`. Thus making `/app` the location of the `pynvml` install

 That seems consistent with the intended usage here

```suggestion
    pip install --no-cache-dir -e . && \
```

Once we do that,